cartography/intel/oci/organizations.py

```diff
@@ -0,0 +1,94 @@
+#Copyright (c) 2020, Oracle and/or its affiliates.
+import re
+
+import logging
+
+import oci
+from oci.exceptions import ConfigFileNotFound, ProfileNotFound, InvalidConfig
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+#def get_caller_identity():
+#    return {}
+
+def get_oci_account_default():
+    try:
+        profile_oci_credentials = oci.config.from_file("~/.oci/config", "DEFAULT")
```

Comment:
[question] not familiar with the oracle sdk , wondering what happens in windows setup,

Reply:
Good question. I looked into it and according to the Oracle SDK setup documentation on Windows they instruct to use PowerShell to create the "~/.oci" directory: https://docs.cloud.oracle.com/en-us/iaas/Content/API/Concepts/sdkconfig.htm

Reply:
Maybe add that information in the documentation